modulefiles/workflow_utils.jet

```diff
@@ -0,0 +1,32 @@
+#%Module#####################################################
+## Workflow Utilities - hera
+#############################################################
+
+# Load cmake
+module load cmake/3.16.1
+
+# Load hpc-stack
+module use /lfs4/HFIP/hfv3gfs/nwprod/hpc-stack/libs/modulefiles/stack
+module load hpc/1.1.0
+
+# Load intel compiler and mpi
+module load hpc-intel/18.0.5.274
+module load hpc-impi/2018.4.274
+
+module load jasper/2.0.25
+module load zlib/1.2.11
+module load png/1.6.35
+
+module load bacio/2.4.1
+module load w3nco/2.4.1
+module load w3emc/2.7.3
+module load sp/2.3.3
+module load ip/3.3.3
+module load nemsio/2.5.2
+module load nemsiogfs/2.5.3
+module load sigio/2.3.2
+module load g2/3.4.1
+module load bufr/11.4.0
+
+module load hdf5/1.10.6
+module load netcdf/4.7.4
```

Comment:
you need to load these after png.
netcdf is used in w3 library.

Reply:
I am testing this item on Jet now.